StaticFusion.h

```diff
@@ -112,6 +112,7 @@ class StaticFusion {
 
 	//Parameters
     float fovh, fovv;							//Field of view of the camera (intrinsic calibration)
+    float fx, fy, cx, cy;
```

Comment:
These variables are only used locally in the constructor and should not be part of the class members. Remove them and only use them locally.